interface/ceed-operator.c

```diff
@@ -170,13 +230,17 @@ int CeedOperatorApply(CeedOperator op, CeedVector in,
   Ceed ceed = op->ceed;
   CeedQFunction qf = op->qf;
 
-  if (op->nfields == 0) return CeedError(ceed, 1, "No operator fields set");
-  if (op->nfields < qf->numinputfields + qf->numoutputfields) return CeedError(
-          ceed, 1, "Not all operator fields set");
-  if (op->numelements == 0) return CeedError(ceed, 1,
-                                     "At least one restriction required");
-  if (op->numqpoints == 0) return CeedError(ceed, 1,
-                                    "At least one non-collocated basis required");
+  if (op->composite) {
```

Comment:
I decided to use the same 'OperatorApply' to make it easier on the user. I don't think it adds much complexity here.